brainfuck2/bf2.kt

```diff
@@ -0,0 +1,116 @@
+import java.io.IOException
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.util.ArrayList
+import java.util.HashMap
+import java.util.Stack
+
+enum class OpT {
+    INC, MOVE, PRINT, LOOP
+}
+
+class Op {
+    var op:OpT
+    var v:Int = 0
+    var loop:List<Op>
+    
+    constructor(_op: OpT, _v: Int) {
+        op = _op
+        v = _v
+        loop = ArrayList<Op>()
+    }
+    
+    constructor(_op: OpT, _l: List<Op>) {
+        op = _op
+        loop = _l
+        v = 0
+    }
+}
```

Comment:
thanks, maybe kotlin have some case class like in scala? to replace this.
https://github.com/kostya/benchmarks/blob/master/brainfuck2/bf.scala#L1